scene/gui/text_edit.cpp

```diff
@@ -6267,6 +6318,49 @@ bool TextEdit::is_highlight_all_occurrences_enabled() const {
 	return highlight_all_occurrences;
 }
 
+void TextEdit::set_use_default_word_separators(bool p_enabled) {
+	if (use_default_word_separators == p_enabled) {
+		return;
+	}
+	use_default_word_separators = p_enabled;
+	text.set_use_default_word_separators(p_enabled);
+	text.invalidate_all_lines();
+}bool TextEdit::is_default_word_separators_enabled() const {
```

Comment:
Accidentally duplicated function

Reply:
Do you mean duplicate of `TextEdit::Text::is_default_word_separators_enabled()`?

Reply:
My bad the formatting was wrong and thought you had accidentally merged the two when copying, was just needing the space